File: src/test/test_files/java/AvoidMultipleFieldsPerClassRuleCodeExample.java
@SuppressWarnings("all")
class LotsOfFields { // Noncompliant
    public int field1;
    public String field2;
    private boolean field3;
    protected char field4;
    public double field5;
    public Field field6;
    public Field field7;
    public Field field8;
    public Field field9;
    public Field field10;
    public Field field11;
    public Field field12;
    public Field field13;
    public Field field14;
}
@SuppressWarnings("all")
class RespectingThreshold { /* Compliant */
    public int field1;
    private String field2;
}

File: src/test/test_files/php/AvoidMultipleFieldsPerClassRuleCodeExample.php
@SuppressWarnings("all")
<?php

class LotsOfFields { // Noncompliant
    public $field1;
    public $field2;
    private $field3;
    protected $field4;
    public $field5;
    public $field6;
    public $field7;
    public $field8;
    public $field9;
    public $field10;
    public $field11;
    public $field12;
    public $field13;
    public $field14;
}
class RespectingThreshold { /* Compliant */
    public $field1;
    private $field2;
}
?>
